ptclient/cts/spi/dx/cl/ctMultiChangeDxa.vue

```diff
@@ -23,27 +23,22 @@
                 >
               </el-form-item>
             </el-form>
-            <DxaHistoryItem :dx="dx"></DxaHistoryItem>
+            <dxa-history-item :dx="dx" />
           </el-card>
         </el-col>
       </el-carousel-item>
     </el-carousel>
   </el-row>
 </template>
 <script>
-import DxaHistoryItem from './ctDxaHistory'
+import DxaHistoryItem from './ctDxaHistory.vue'
```

Comment:
SFC's should include the `vue` extension when importing other components. This helps peek and jump resolution in vscode.